packages/js/product-editor/src/components/variations-table/variation-actions-menus/test/variation-quick-update-menu-item.test.tsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { render, fireEvent } from '@testing-library/react';
+import React, { createElement } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getGroupName, getMenuItem } from '..';
+import {
+	MULTIPLE_UPDATE,
+	SINGLE_UPDATE,
+	VARIATION_ACTIONS_SLOT_NAME,
+} from '../constants';
+
+describe( 'getMenuItem', () => {
+	let onClickMock: jest.Mock;
+	beforeEach( () => {
+		onClickMock = jest.fn();
+	} );
+
+	it( 'should render the menuItem content', () => {
+		const { queryByText } = render(
+			getMenuItem( <button>My button</button>, onClickMock )
+		);
+		expect( queryByText( 'My button' ) ).toBeInTheDocument();
+	} );
+
+	it( 'should call onClick when the menuItem is clicked', async () => {
+		const { getByRole } = render(
+			getMenuItem( <button>My button</button>, onClickMock )
+		);
+		await fireEvent.click( getByRole( 'button', { name: 'My button' } ) );
+		expect( onClickMock ).toHaveBeenCalled();
+	} );
```

Comment:
It is confusing having a button in this test, as the `MenuItem` itself is a `<button>` with `role="menuitem"`. I think we should be using those when testing.

Reply:
I modified the test in the commit [b54966b](https://github.com/woocommerce/woocommerce/pull/43441/commits/b54966b28828d255b3c984f9e6500a24c66f2223)